engine/src/flutter/shell/platform/android/io/flutter/plugin/view/SensitiveContentPlugin.java

```diff
@@ -0,0 +1,73 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugin.view;
+
+import android.app.Activity;
+import android.view.View;
+import androidx.annotation.NonNull;
+import io.flutter.embedding.engine.systemchannels.SensitiveContentChannel;
+import io.flutter.plugin.common.MethodChannel;
+
+/**
+ * {@link SensitiveContentPlugin} is the implementation of all functionality needed to set
+ * the content sensitivity level of a native Flutter Android {@code View}.
+ *
+ * <p>This plugin handles requests for setting content sensitivity sent by the {@link
+ * io.flutter.embedding.engine.systemchannels.SensitiveContentChannel}.
+ */
+public class SensitiveContentPlugin
+    implements SensitiveContentChannel.SensitiveContentMethodHandler {
+
+  private final Activity mflutterActivity;
+  private final SensitiveContentChannel mSensitiveContentChannel;
+
+  public SensitiveContentPlugin(
+      @NonNull Activity flutterActivity, @NonNull SensitiveContentChannel sensitiveContentChannel) {
+    mflutterActivity = flutterActivity;
+    mSensitiveContentChannel = sensitiveContentChannel;
+
+    mSensitiveContentChannel.setSensitiveContentMethodHandler(this);
+  }
+
+  /**
+   * Sets content sensitivity level of the Android {@code View} with the specified {@code
+   * flutterViewId} to the level specified by {@contentSensitivity}.
+   */
+  @Override
+  public void setContentSensitivity(
+      @NonNull int flutterViewId,
+      @NonNull int contentSensitivity,
+      @NonNull MethodChannel.Result result) {
+    final View flutterView = mflutterActivity.findViewById(flutterViewId);
+    if (flutterView == null) {
+      result.error("error", "Requested Flutter View to set content sensitivty of not found.", null);
+    }
+
+    // Set contentSensitivity on the requested View.
+    flutterView.setContentSensitivity(contentSensitivity);
```

Comment:
Where are you protecting against android api levels that dont have this method? 

If you are not I would look at where we landed for @justinmc scribe pr.

Reply:
This is the PR: https://github.com/flutter/engine/pull/52943